kms/key/kms.go

```diff
@@ -0,0 +1,84 @@
+package main
+
+import (
+	"github.com/acorn-io/aws/kms/key/props"
+	"github.com/acorn-io/services/aws/libs/common"
+	"github.com/aws/aws-cdk-go/awscdk/v2"
+	"github.com/aws/aws-cdk-go/awscdk/v2/awsiam"
+	"github.com/aws/aws-cdk-go/awscdk/v2/awskms"
+	"github.com/aws/constructs-go/constructs/v10"
+	"github.com/aws/jsii-runtime-go"
+	"github.com/sirupsen/logrus"
+)
+
+func newKMSKeyStack(scope constructs.Construct, id string, props *props.KMSKeyStackProps) (awscdk.Stack, error) {
```

Comment:
Should be `NewKMSKeyStack`. It's weird to export a function in `main` but that's the standard we've been following.

Reply:
Forgot to fix this earlier. Fixed it now.